python_modules/libraries/dagster-components/setup.py

```diff
@@ -36,7 +36,7 @@ def get_version() -> str:
     ],
     packages=find_packages(exclude=["dagster_components_tests*", "examples*"]),
     install_requires=[
-        f"dagster{pin}",
+        "dagster",
```

Comment:
we should have some sort of pin here. We'll have to manually manage it.

Reply:
Adding 1.9.5